styles.css

```diff
@@ -1,316 +1,297 @@
-  
-  }
+.hero-h1 {
+  background-image: url(./img/color.jpg);
+  font-size: 5rem;
+  font-weight: 900;
+  width: 60%;
+  margin: 0;
+  font-family: 'Cocogoose', sans-serif;
+  line-height: 1;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  background-clip: text;
+}
+
+.hero-p {
+  font-size: 12px;
+  width: 30%;
+  height: 20%;
+  border: 2px solid white;
+  background-color: rgb(238, 235, 235, 0.5);
+  padding: 5px 10px;
+  color: rgb(102, 104, 104);
+  font-weight: 600;
+  margin-top: 20px;
+}
 
-  .hero-h1 {
-    background-image: url(./img/color.jpg);
-    font-size: 5rem;
-    font-weight: 900;
-    width: 60%;
-    margin: 0;
-    font-family: 'Cocogoose', sans-serif;
-    line-height: 1;
-    -webkit-background-clip: text;
-    -webkit-text-fill-color: transparent;
-    background-clip: text;
-    text-fill-color: transparent;
-  }
-
-  .hero-p{
-    font-size: 12px;
-    width: 30%;
-    height: 20%;
-    border: 2px solid white;
-    background-color: rgb(238, 235, 235,0.5);
-    padding: 5px 10px;
-    color: rgb(102, 104, 104);
-    font-weight: 600;
-    margin-top: 20px;
-  }
+.hero-h3 {
+  font-family: 'Cocogoose', sans-serif;
+  font-size: 40px;
+  font-weight: 800;
+  width: 50%;
+  margin-bottom: 0;
+  color: #2e2e2e;
+}
 
-  .hero-h3{
-    font-family: 'Cocogoose',sans-serif;
-    font-size: 40px;
-    font-weight: 800;
-    
-    width: 50%;
-    margin-bottom: 0;
-    color: #2e2e2e;
-  }
+.bottom-p {
+  font-family: 'Cocogoose', sans-serif;
+  font-size: 20px;
+  font-weight: 600;
+  margin: 0;
+  color: #565758;
+  padding-bottom: 50px;
+}
 
-  .bottom-p{
-    font-family: 'Cocogoose',sans-serif;
-    font-size: 20px;
-    font-weight: 600;
-    margin: 0;
-    color: #565758;
-    padding-bottom: 50px;
-  }
+/* Main Program section */
 
+#program {
+  height: auto;
+  text-align: center;
+  padding: 100px 200px;
+  background-color: #272a31;
+  position: relative;
+  z-index: 1;
+}
 
-  /* Main Program section */
+#program::before {
+  content: "";
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-image: radial-gradient(black 1px, transparent 1px);
+  background-size: 10px 10px;
+  pointer-events: none;
+  z-index: -1;
+}
 
+.program-h2 {
+  color: white;
+}
 
-  #program {
-    height: auto;
-    text-align: center;
-    padding: 100px 200px;
-    background-color: #272A31;
-    position: relative;
-    z-index: 1;
-  }
-  
-  #program::before {
-    content: "";
-    position: absolute;
-    top: 0;
-    left: 0;
-    right: 0;
-    bottom: 0;
-    background-image: radial-gradient(black 1px, transparent 1px);
-    background-size: 10px 10px;
-    pointer-events: none;
-    z-index: -1; 
-  }
+.program-hr {
+  width: 50px;
+  border-color: red;
+}
 
-  .program-h2 {
-    color: white;
-  
-  }
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
+  gap: 5px;
+  margin: 100px 0;
+  overflow-x: hidden;
+}
```

Comment:
- [ ] [REQUIRED] Good job so far! Please give `hover effect` to each card like a [Behance Design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 🙏🏼 

![image](https://github.com/Xghayor/Music-Festival/assets/63915024/e8704d25-4c67-4aa6-b952-9678a9bfa0a3)